### PR TITLE
Remove dummy box to solve illegal case in geometry

### PR DESCRIPTION
**Pull request by frmanso**
Following JIRA request https://alice.its.cern.ch/jira/browse/O2-1046 , the dummy box is removed.

### CODE REVIEW

File: Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx [hunk 1]
```diff
@@ -532,12 +530,29 @@ void HeatExchanger::createManifold(Int_t disk)
   TGeoRotation* rcoverBodyStep = new TGeoRotation(Form("rcoverBodyStep4MF%d", disk), 45, 0, 0);
   rcoverBodyStep->RegisterYourself();
 
-  TGeoCompositeShape* coverBodyStep4rotated = new TGeoCompositeShape(Form("coverBodyStep4rotatedMF%d", disk), Form("dummy + coverBodyStep4MF%d:rcoverBodyStep4MF%d", disk, disk));
+  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  //Add 01/07/2020
+  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (sawenzel) [inline Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx:535]: do you need these date + // annotations? Probably a detailed explanation (in the git commit message) would be better.

File: Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx [hunk 2]
```diff
@@ -532,12 +530,29 @@ void HeatExchanger::createManifold(Int_t disk)
...
+  //
+  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+  //TGeoCompositeShape* coverBodyStep4rotated = new TGeoCompositeShape(Form("coverBodyStep4rotatedMF%d", disk), Form("dummy + coverBodyStep4MF%d:rcoverBodyStep4MF%d", disk, disk));
 
   for (Int_t i = 0; i < 4; ++i)
     tcoverBodyStep[i]->RegisterYourself();
 
-  TGeoCompositeShape* shapeStep = new TGeoCompositeShape(Form("shapeStepMF%d", disk), Form("coverBodyStep1MF%d:tcoverBodyStep1MF%d + coverBodyStep2MF%d:tcoverBodyStep2MF%d + coverBodyStep3MF%d:tcoverBodyStep3MF%d + coverBodyStep4rotatedMF%d:tcoverBodyStep4MF%d", disk, disk, disk, disk, disk, disk, disk, disk));
+  //TGeoCompositeShape* shapeStep = new TGeoCompositeShape(Form("shapeStepMF%d", disk), Form("coverBodyStep1MF%d:tcoverBodyStep1MF%d + coverBodyStep2MF%d:tcoverBodyStep2MF%d + coverBodyStep3MF%d:tcoverBodyStep3MF%d + coverBodyStep4rotatedMF%d:tcoverBodyStep4MF%d", disk, disk, disk, disk, disk, disk, disk, disk));
```
Comment (sawenzel) [inline Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx:548]: I would avoid leaving these old code sections commented

File: Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx (hunk 1 above)
Comment (frmanso) [reply Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx]: I removed these uneeded lines

File: Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx (hunk 2 above)
Comment (frmanso) [reply Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx]: Yep, it's done with the new version

**Pull request merged**